doc/sphinx/Makefile

```diff
@@ -60,22 +75,27 @@ woke-install:
 
 pa11y-install:
 	@type $(PA11Y) >/dev/null 2>&1 || { \
-			echo "Installing \"pa11y\" from npm... \n"; \
+			echo "Installing \"pa11y-ci\" from npm... \n"; \
 			mkdir -p $(SPHINXDIR)/node_modules/ ; \
-			npm install --prefix $(SPHINXDIR) pa11y; \
+			npm install --prefix $(SPHINXDIR) pa11y-ci; \
 		}
 
 install: $(VENVDIR)
+	sudo apt install -y $(DEPS)
+
+run: install configure
+	. $(VENV); cd $(WORKSPHINX); sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
 
-run: install
-	. $(VENV); sphinx-autobuild -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS)
+$(WORKSPHINX)/_build/index.html:
+	cd $(WORKSPHINX); \
+	. $(VENV); \
+	$(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w $(SPHINXDIR)/warnings.txt $(SPHINXOPTS)
 
-# Doesn't depend on $(BUILDDIR) to rebuild properly at every run.
-html: install
-	. $(VENV); $(SPHINXBUILD) -b dirhtml "$(SOURCEDIR)" "$(BUILDDIR)" -w $(SPHINXDIR)/warnings.txt $(SPHINXOPTS)
+# Make sure that HTML docs are built at least once
+html: install configure $(WORKSPHINX)/_build/index.html
```

Comment:
Reading the comment above, it looks to be rebuilt on purpose (otherwise a manual `make html` will not work on subsequent runs). While I get it that this costs us in CI, I'm not sure it's worth the potential confusion when it doesn't work?

Reply:
HTML docs need to only be built once per run, no need to rebuild them for different targets. So why would we need to rebuild them each time?